.github/workflows/update-formula.yml

```diff
@@ -0,0 +1,33 @@
+name: create PR to update formula
+
+on:
+  push:
+    branches:
+      - imgpkg-homebrew
+permissions:
+  pull-requests: write
+  contents: none
+  actions: none
+  checks: none
+  deployments: none
+  issues: none
+  packages: none
+  repository-projects: none
+  security-events: none
+  statuses: none
+jobs:
+  update-imgpkg-homebrew:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+      with:
+        repository: vmware-tanzu/homebrew-carvel
+        ref: develop
+    - name: pull-request
+      uses: repo-sync/pull-request@65194d8015be7624d231796ddee1cd52a5023cb3
```

Comment:
since this is just creating a PR, can we inline hub commands here to have less dependencies on 3rd party actions.

Reply:
updated: github PR has been inlined

This should be merge ready and doesn't require any creds added to make it work